mmcv/cnn/rfsearch/search.py

```diff
@@ -0,0 +1,240 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import logging
+import os
+from typing import Dict, Optional
+
+import torch  # noqa
+import torch.nn as nn
+
+import mmcv
+from mmcv.cnn.rfsearch.utils import get_single_padding, write_to_json
+from mmcv.runner import HOOKS, Hook
+from .operator import BaseConvRFSearchOp, Conv2dRFSearchOp  # noqa
+
+logging.basicConfig(
+    format='[%(asctime)s-%(filename)s-%(levelname)s:%(message)s]',
+    level=logging.NOTSET,
+    datefmt='%Y-%m-%d%I:%M:%S %p',
+)
+logger = logging.getLogger('Searcher')
+logger.setLevel(logging.ERROR)
```

Comment:
The final comment is that can we replace those lines with `logger = get_logger('mmcv')`?

```python
from mmcv.utils import get_logger
logger = get_logger('mmcv')
```

Reply:
I have replace the logger following your suggestions.